include/dsn/utils/latency_tracer.h

```diff
@@ -53,14 +55,19 @@ namespace utils {
  *    |         |           |         |
  *  start---->stageA----->stageB---->end
  *
- * "request.tracer" will record the time duration among all tracepoints.
+ * "request.tracer" will record the time duration among all trace points.
 **/
-extern bool FLAGS_enable_latency_tracer;
+DSN_DECLARE_bool(enable_latency_tracer);
 
 class latency_tracer
 {
 public:
-    latency_tracer(const std::string &name);
+    //  threshold < 0: don't dump any trace points
+    //  threshold = 0: dump all trace points
+    //  threshold > 0: dump the trace point when time_used > threshold
+    latency_tracer(const std::string &name, const uint64_t threshold = 0);
```

Comment:
```suggestion
    latency_tracer(const std::string &name, uint64_t threshold = 0);
```

`const` for value is redundant